src/TwigRenderer.php

```diff
@@ -36,16 +39,16 @@ class TwigRenderer implements TemplateRendererInterface
     private $suffix;
 
     /**
-     * @var TwigFilesystem
+     * @var FilesystemLoader
      */
     protected $twigLoader;
 
     /**
-     * @var TwigEnvironment
+     * @var Environment
      */
     protected $template;
 
-    public function __construct(TwigEnvironment $template = null, string $suffix = 'html')
+    public function __construct(Environment $template = null, string $suffix = 'html')
```

Comment:
We'll need to revert this to the non-namespaced version, for reasons listed in the factory earlier. That also applies to the `$template` and `$twigLoader` properties, as they are not private.